geode-wan/src/distributedTest/java/org/apache/geode/internal/cache/wan/concurrent/ConcurrentParallelGatewaySenderDUnitTest.java

```diff
@@ -633,6 +663,28 @@ public void testPartitionedParallelPropagationHA() throws Exception {
     vm7.invoke(() -> WANTestBase.validateParallelSenderQueueAllBucketsDrained("ln"));
   }
 
+  private void waitForRegionToBeReadyOnServers(String regionName, int expectedNumRegions,
+      VM... vms) {
+    int numRegionsReady = 0;
```

Comment:
`expectedNumRegions` isn't needed as an argument. You should be able to declare it locally
```
int expectedNumRegions = vms.length
```

Reply:
Another suggestion: `expectedNumRegions` and `numRegionsReady` are really referring the servers that are ready (with the given region). So maybe rename to `expectedVmsWithRegion` and `numVmsReady` to be more indicative of how they're used.

Reply:
Great suggestions, thank you!